nixos/modules/services/web-apps/lubelogger.nix

```diff
@@ -0,0 +1,94 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.lubelogger;
+in
+{
+  options = {
+    services.lubelogger = {
+      enable = mkEnableOption "Lubelogger, a self-hosted, open-source, web-based vehicle maintenance and fuel milage tracker";
+
+      package = mkPackageOption pkgs "lubelogger" { };
+
+      dataDir = mkOption {
+        description = "Path to Lubelogger config and metadata inside of /var/lib.";
+        default = "lubelogger";
+        type = types.str;
+      };
+
+      port = mkOption {
+        description = "The TCP port Lubelogger will listen on.";
+        default = 5000;
+        readOnly = true; # Lubelogger does not allow you to configure the port it runs on
+        type = types.port;
+      };
+
+      user = mkOption {
+        description = "User account under which Lubelogger runs.";
+        default = "audiobookshelf";
```

Comment:
Should this also be lubelogger?

Reply:
yes it should be. thought i caught them all but i guess i missed one